hazelcast-jet-core/src/main/java/com/hazelcast/jet/impl/execution/init/EdgeDef.java

```diff
@@ -114,6 +114,10 @@ Address getDistributedTo() {
         return distributedTo;
     }
 
+    boolean isDistributed() {
```

Comment:
I specifically didn't want to create this method because of the distributed-to-one edge type - it won't strike your eye that there are two types of distributed edges, especially when the distributed-to-one is rarely used. Maybe we can create `isLocal` method instead, it seems that there are more instances of `!edge.isDistributed()` than without the negation.